src/components/page/index.jsx

```diff
@@ -16,6 +16,9 @@ class Page extends React.PureComponent {
       tables,
       image,
       WidgetTag,
+      widget: {
+        layout: { row: widgetRow },
+      },
```

Comment:
This is what's causing the build to fail: not all pages have widgets (and not all widgets have layout fields on them for that matter), so sometimes widget is null, so sometimes this line is attempting to destructure null which is a non-starter.  Probably the best way to address this would be to break this line into multiple steps:
```
const { layout } = widget || {};
const { row: widgetRow } = layout || {}; 
```
And then deal with the fact that sometimes widgetRow will be null.